zipkin-lens/package.json

```diff
@@ -26,12 +26,9 @@
     "@material-ui/pickers": "^3.1.1",
     "@material-ui/styles": "^4.1.1",
     "@reduxjs/toolkit": "^1.3.4",
-    "@testing-library/jest-dom": "^5.5.0",
-    "@testing-library/react": "^10.0.2",
-    "@testing-library/react-hooks": "^3.2.1",
-    "@testing-library/user-event": "^10.0.2",
-    "@types/enzyme": "^3.10.5",
-    "@types/enzyme-adapter-react-16": "^1.0.6",
+    "@testing-library/jest-dom": "6.1.5",
+    "@testing-library/react": "12.1.5",
```

Comment:
This is not the latest version.
testing-library/react  13+ requires React v18, but we still use React v16, so I could not update to the latest version (14.1.2).

https://www.npmjs.com/package/@testing-library/react